src/components/learner-credit-management/AssignmentRowActionTableCell.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Icon, OverlayTrigger, Tooltip } from '@edx/paragon';
+import { Mail, DoNotDisturbOn } from '@edx/paragon/icons';
+
+const AssignmentRowActionTableCell = ({ row }) => (
+  <div className="d-flex">
+    {row.original.state === 'allocated' && (
+      <OverlayTrigger
+        key="Remind"
+        placement="top"
+        overlay={<Tooltip variant="light" id="tooltip-remind">Remind learner</Tooltip>}
+      >
+        <Icon
+          className="ml-auto mr-0"
+          src={Mail}
+          // eslint-disable-next-line no-console
+          onClick={() => console.log(`Reminding ${row.original.uuid}`)}
+          data-testid={`remind-assignment-${row.original.uuid}`}
+        />
+      </OverlayTrigger>
+    )}
+    <div />
+    <OverlayTrigger
+      key="Cancel"
+      placement="top"
+      overlay={<Tooltip variant="light" id="tooltip-cancel">Cancel assignment</Tooltip>}
+    >
+      <Icon
+        className="ml-4.5 mr-2 text-danger-500"
+        src={DoNotDisturbOn}
+        // eslint-disable-next-line no-console
+        onClick={() => console.log(`Canceling ${row.original.uuid}`)}
+        data-testid={`cancel-assignment-${row.original.uuid}`}
+      />
+    </OverlayTrigger>
+  </div>
+);
+
+AssignmentRowActionTableCell.propTypes = {
+  row: PropTypes.shape({
+    original: PropTypes.shape({
+      uuid: PropTypes.string.isRequired,
+      learnerEmail: PropTypes.string.isRequired,
```

Comment:
nit: `learnerEmail` will not always be available/returned once the assignments stuff is integrated with the GDPR/PII retirement pipeline for when user's request to have their data scrubbed from our records.

Related, if `learnerEmail` isn't actually used within this component (same with `contentKey`), should they be excluded from the prop types for this component?

Reply:
oh interesting! This is copied from the other cell component so will we need to account for this there?

Reply:
and yeah, relic of copy paste tho and I'll remove

Reply:
> This is copied from the other cell component so will we need to account for this there?

Ah, yeah... it was recently made consistent across all `learner-credit-management` components in a recently merged commit. Current state on `master` does not depict required `learnerEmail` anywhere.